### DISCUSSION TITLE
USB trigger with live stream

### DISCUSSION THREAD

**Discussion by gigrant85** [gopro/labs#869] [Q&A]
Hi is it possible to create a qr code with usb trigger and livestream.

**Comment by dnewman-gpsw** [comment]
Yes. What have you tried?  https://gopro.github.io/labs/control/rtmp/

**Comment by gigrant85** [comment]
Hi David,

Thanks for getting back to me. I’ve only just found out about GoPro labs.
Trying to set up on my rally car. Managed to wt the usb trigger to work.
But wondered if could usb trigger to automatically start live stream. I’ve
no really idea how to do it. To be honest



Kind Regards
Greg

**Comment by dnewman-gpsw** [comment]
You will have to experimented first, work out how to start a Livestream with a QR code.  Once this is working the same QR Code, can be made into a BOOT command, so that it is the default active on boot.  Separately there is the WAKE command, that make the camera boot when power is added to USB.  With these three features combined, using Labs firmware, you can have a camera auto boot and start livestream as soon as you power up your rally car.

**Comment by gigrant85** [comment]
Hi David,
Just away to experiment with it just now. Quick question. If I manage to
make it work to live stream on boot. Would there also be a way for it to
record at same time. Or record on the failure of connecting to network for
live streaming?

Kind Regards
Greg

**Reply by dnewman-gpsw** [reply]
Record at the same time is a standard option for Livestream.  Retry should be doable with HERO11 or HERO12, or the next Labs firmware for HERO10.  I so rarely use Livestreaming so I have limited personal experience with this feature operated through Labs. This is why you need to experiment.
